From: Ben Hutchings <ben@decadent.org.uk>
Subject: mm: avoid ABI change in 3.2.33
Date: Sun, 18 Nov 2012 07:07:43 +0000
Forwarded: not-needed

Don't let genksyms see the new #include, as this changes symbol versions.

---
Index: linux-3.2.46/mm/rmap.c
===================================================================
--- linux-3.2.46.orig/mm/rmap.c	2013-07-26 18:54:01.000000000 +0000
+++ linux-3.2.46/mm/rmap.c	2013-07-26 19:25:11.000000000 +0000
@@ -56,7 +56,9 @@
 #include <linux/mmu_notifier.h>
 #include <linux/migrate.h>
 #include <linux/hugetlb.h>
+#ifndef __GENKSYMS__
 #include <linux/backing-dev.h>
+#endif
 
 #include <asm/tlbflush.h>
 
